verification/src/transaction_verifier.rs

```diff
@@ -40,13 +40,10 @@ where
 
 pub struct TransactionVerifier<'a, M> {
     pub version: VersionVerifier<'a>,
-    pub null: NullVerifier<'a>,
     pub empty: EmptyVerifier<'a>,
     pub maturity: MaturityVerifier<'a>,
     pub capacity: CapacityVerifier<'a>,
-    pub duplicate_inputs: DuplicateInputsVerifier<'a>,
     pub duplicate_deps: DuplicateDepsVerifier<'a>,
```

Comment:
Should also remove this?

Reply:
`DuplicateDepsVerifier` is needed here, because `resolve_transaction` fn only checks the duplicate inputs with the `seen_inputs` set.